doc/api/crypto.md

```diff
@@ -1821,6 +1821,9 @@ Creates and returns a new key object containing a private key. If `key` is a
 string or `Buffer`, `format` is assumed to be `'pem'`; otherwise, `key`
 must be an object with the properties described above.
 
+If the private key is encrypted, a `passphrase` must be specified. The length
+of the passphrase is limited.
```

Comment:
Maybe be less coy about the limit? You could say something like "on the order of 1024 bytes."